src/deluge/gui/menu_item/midi/device_specific/lumi_keys.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2023 Aria Burrell (litui)
+ *
+ * This file is part of The Synthstrom Audible Deluge Firmware.
+ *
+ * The Synthstrom Audible Deluge Firmware is free software: you can redistribute it and/or modify it under the
+ * terms of the GNU General Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program.
+ * If not, see <https://www.gnu.org/licenses/>.
+*/
+
+class LumiKeys {}
```

Comment:
this file seems empty?

Reply:
This file shouldn't exist... I'll see where it's coming from.